modules/compose/Makefile

```diff
@@ -0,0 +1,44 @@
+DOCKER_NETWORK ?= $(COMPOSE_PROJECT_NAME)_default
+
+DOCKER_MONITOR_TABLE ?= 'table {{.Name}}\t{{.CPUPerc}}\t{{.MemUsage}}\t{{.NetIO}}\t{{.BlockIO}}'
+
+DOCKER_CMD ?= docker run -i --rm --network=$(DOCKER_NETWORK)
+
+SSH_EXEC ?= docker exec -it
```

Comment:
This is declared in a couple of places. One will mask the other. Also, see other comments about name of this variable.